userprefs.php

```diff
@@ -181,7 +184,7 @@ function do_font_sample_update(font_index, size_index, layout) {
 
 output_tab_bar($tabs, $selected_tab, "tab", "origin=" . urlencode($origin));
 
-echo "<p>" . _("Click the ? for help on that specific preference.") . "</p>";
+echo "<p>" . _("Hover over a setting to get more information about it.") . "</p>";
```

Comment:
hover may be hard to understand in the context of a touch device. Also, to me at least, the hover behavior was a little obtrusive navigating around my user preferences. Probably wouldn't think twice about it unless observing the behavioral differences though.

Reply:
I tried it on my (Android) phone. Press-and-hold worked just fine. On my iPads (2017 and 2024), press-and-hold does not work. Turned out that just a quick tap did work for the iPads. I don't have an iPhone to be able to tell what works on them.

Reply:
Yeah, it's not ideal but I couldn't come up with a better way that made them accessible without being obtrusive. One of the reasons I put them over the labels instead of over the fields themselves.

I'm open to other ideas for how to present this info!

Reply:
I tell you what. I'm going to merge this in with the knowledge that we might need to iterate over how to best display this on the page -- but that we'll display it on the page somehow rather than in a new page, so it's a matter of adjusting the JS display code. This unblocks me from putting up a second PR that relies on this code being in.